templates/cli/src/Parser.php.twig

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace {{ spec.title | caseUcfirst }};
+
+use jc21\CliTable;
+use jc21\CliTableManipulator;
+
+class Manipulators extends CliTableManipulator {
```

Comment:
This is non-standard in PHP. Use 1 file for 1 class.

Reply:
Done